src/interfaces/filing-interfaces/incorporation-filing-interface.ts

```diff
@@ -1,21 +0,0 @@
-import { IncorporationApplicationIF } from '@/interfaces/'
```

Comment:
The content of this file was merged into src/interfaces/filing-interfaces/incorporation-interfaces.ts below.